frontend/src/metabase/sharing/components/testSetup.js

```diff
@@ -0,0 +1,126 @@
+import fetchMock from "fetch-mock";
```

Comment:
Wdyt about organizing it according to the [doc](https://www.notion.so/metabase/Testing-paid-features-49e03e1b2a3b41ffb4ebdc7616a93d90)?
It would be better because of https://github.com/metabase/metabase/pull/32164

Reply:
can do. I'll just have to restructure the directory a bit because there are a lot of components in this directory